src/API/WP/OAuthService.php

```diff
@@ -159,16 +159,57 @@ public function revoke_wpcom_api_auth(): string {
 		$request = $this->container->get( Jetpack::class )->remote_request( $args );
 
 		if ( is_wp_error( $request ) ) {
+
+			/**
+			 * @event revoke_wpcom_api_auth
+			 */
```

Comment:
Can we add here the property directives? Also, I miss the event description.

Reply:
Thanks I add them here: [fb8b49c](https://github.com/woocommerce/google-listings-and-ads/pull/2452/commits/fb8b49cf7b43daf59cac9f48b00d162d432c4943)